recipes-core/customizations-example/files/board-configuration

```diff
@@ -69,7 +69,7 @@ def initAruinoPins():
 
 
 def initExternalSerialMode():
-    command = 'grep -a -o -P "IOT2050 \w*" /proc/device-tree/model'
+    command = 'grep -a -o -P "IOT2050 [\w\s]+" /proc/device-tree/model'
```

Comment:
Why need change this?

Reply:
This is intended to distinguish between `Advanced`  and `Advanced PG2`.
and do the  the serial port reset for PG2 devices